uberfire-extensions/uberfire-wires/uberfire-wires-core/uberfire-wires-core-grids/src/main/java/org/uberfire/ext/wires/core/grids/client/model/GridData.java

```diff
@@ -295,6 +295,39 @@ void collapseCell(final int rowIndex,
     void expandCell(final int rowIndex,
                     final int columnIndex);
 
+    /**
+     * Update width for column with {@link GridColumn.ColumnWidthMode#auto}
```

Comment:
Sorry, an _English_ observation (and I do apologise). Would "Updates the width of columns with {@link...}" read better (when I first read this I had to double check this was not on the `GridColumn` interface).